controllers/sr_controller/sr/robot/robot.py

```diff
@@ -52,6 +52,13 @@ def display_info(self) -> None:
             mode=self.mode,
         ))
 
+        user_version_path = path.join(self.usbkey, '.user-rev')
+        if path.exists(user_version_path):
+            with open(user_version_path) as f:
+                user_version = f.read().strip()
+
+            print("User code: {}".format(user_version))  # noqa:T001
```

Comment:
Might be nice to combine this with the above print? Just to reduce noise

Reply:
I wondered about that, though not for noise reasons -- instead for clarity while #128 is outstanding.

Reply:
365931f